frontend/src/pages/org/archived-item-detail/ui/qa.ts

```diff
@@ -488,23 +491,23 @@ export class ArchivedItemDetailQA extends TailwindElement {
               const finishedQARuns = qaRuns.filter(({ state }) =>
                 finishedCrawlStates.includes(state),
               );
+              const latestFinishedSelected =
+                this.qaRunId === finishedQARuns[0]?.id;
 
-              if (!finishedQARuns.length) {
-                return nothing;
-              }
-
+              const finishedAndRunningQARuns = qaRuns.filter(
+                ({ state }) =>
+                  finishedCrawlStates.includes(state) ||
+                  QA_RUNNING_STATES.includes(state),
+              );
               const mostRecentSelected =
-                this.mostRecentNonFailedQARun &&
-                this.mostRecentNonFailedQARun.id === this.qaRunId;
-              const latestFinishedSelected =
-                this.qaRunId === finishedQARuns[0].id;
+                this.qaRunId === finishedAndRunningQARuns[0]?.id;
 
               return html`
                 <div>
                   <sl-tooltip
                     content=${mostRecentSelected
                       ? msg(
-                          "You’re viewing the latest results from a finished analysis run.",
+                          "You’re viewing the latest results from an analysis run.",
```

Comment:
Maybe this tooltip isn't needed any more? Or could be:
```suggestion
                          "You’re viewing the latest analysis run results.",
```